plugins/cinema4d/res/c4d_symbols.h

```diff
@@ -8,5 +8,6 @@ enum
 	CONSTRAIN_SCALE = 10006,
 	CONSTRAIN_ROT = 10007,
 
-	IK_TAG = 1019561
+	IK_TAG = 1019561,
```

Comment:
These should be able to be cleaned up to use the c4d globals already I think.

Reply:
I just found the globals.
Would you prefer something like `IK_TAG = c4d.Tcaik` or going straight to `c4d.BaseTag(c4d.Tcaik)`?

Especially for the constraint stuff, since there will be multiple duplicates. I don't know how important that would be from a style and a performance perspective. So I personally would just replace the integers with the c4d globals.
What is your opinion on that?